k8s/kubectl_configure.bzl

```diff
@@ -0,0 +1,31 @@
+# Copyright 2018 The Bazel Authors. All rights reserved.
```

Comment:
lets move this file to toolchains/kubectl so the reference to :BUILD.tpl can be relative

Reply:
Moved kubectl_configure.bzl to toolchains/kubectl.
But the reference to BUILD.tpl still has to be absolute. If (in repository_ctx.template) a string or a path object is used to specify the template file, then it will be relative to the local_k8s_config repo. If a Label is used, then an absolute path must be specified.